trulens/nn/models/keras.py

```diff
@@ -65,13 +97,18 @@ def __init__(
             modified. By default `False`.
         """
         self.keras = import_keras_backend()
+        self.tf = import_tensorflow()
+        self.tfhub = import_tfhub_deps()
```

Comment:
Is this saying we only support tfhub models? (this fails if tfhub not installed)

Reply:
Ohh, nevermind, read it wrong.